Explorer/Assets/DCL/MarketplaceCredits/MarketplaceCreditsMenuController.cs

```diff
@@ -54,11 +54,11 @@ public partial class MarketplaceCreditsMenuController : ControllerBase<Marketpla
         private readonly ISharedSpaceManager sharedSpaceManager;
         private readonly FeatureFlagsCache featureFlagsCache;
 
-        private MarketplaceCreditsWelcomeSubController marketplaceCreditsWelcomeSubController;
-        private MarketplaceCreditsVerifyEmailSubController marketplaceCreditsVerifyEmailSubController;
-        private MarketplaceCreditsGoalsOfTheWeekSubController marketplaceCreditsGoalsOfTheWeekSubController;
-        private MarketplaceCreditsWeekGoalsCompletedSubController marketplaceCreditsWeekGoalsCompletedSubController;
-        private MarketplaceCreditsProgramEndedSubController marketplaceCreditsProgramEndedSubController;
+        [CanBeNull] private MarketplaceCreditsWelcomeSubController marketplaceCreditsWelcomeSubController;
```

Comment:
Please use C# notations. Include the file `csc.rsp`: `-nullable:enable`

Reply:
Done: https://github.com/decentraland/unity-explorer/pull/4048/commits/3590bda9daf9405c23a5fc89c6dc3417ed01e040